addons/binding/org.openhab.binding.pioneeravr/src/main/java/org/openhab/binding/pioneeravr/protocol/utils/VolumeConverter.java

```diff
@@ -15,72 +15,71 @@
  * @author Antoine Besnard - Initial contribution
  */
 public final class VolumeConverter {
-
-    private static final String IP_CONTROL_VOLUME_FORMAT = "000";
-    private static final String IP_CONTROL_VOLUME_DEFAULT_VALUE = "000";
-
-    private static final double MAX_IP_CONTROL_VOLUME = 184;
-    private static final double MIN_DB_VOLUME = 80;
-
-    private static final DecimalFormat FORMATTER = new DecimalFormat(IP_CONTROL_VOLUME_FORMAT);
-
+    
+    private static final String[] IP_CONTROL_VOLUME_FORMAT = { "000", "00", "00", "00" };
```

Comment:
Could you give some more background of the values, where do they come from, maybe you could also add some documentation?